runner/Makefile

```
@@ -5,6 +5,7 @@ TAG ?= latest
 TARGETPLATFORM ?= $(shell arch)
 
 RUNNER_VERSION ?= 2.294.0
+RUNNER_CONTAINER_HOOKS_VERSION ?= 0.1.1
```

Nit but

```suggestion
RUNNER_CONTAINER_HOOKS_VERSION ?= 0.1.2
```

(Ideally, we need a mechanism to sync this with the default value embedded in the dockerfile!

---

Great catch thank you!